### DISCUSSION TITLE
Offline Request Queue

### DISCUSSION THREAD

**Discussion by jonathanadams** [trpc/trpc#4759] [Q&A]
I'm currently exploring tooling for use in a react native application. 

I am developing an app whereby the majority of the time it will be online and able to connect to the api server but if there is a network outage then I still need the app to be usable (I.e queuing requests to be sent).

Does anyone have any ideas on the best way to do this?

**Comment by sachinraja** [comment]
See this: https://tkdodo.eu/blog/offline-react-query
